pkg/apis/kops/validation/cluster.go

```diff
@@ -118,10 +118,6 @@ func validateEtcdMemberUpdate(fp *field.Path, obj kops.EtcdMemberSpec, status *k
 		allErrs = append(allErrs, field.Forbidden(fp.Child("kmsKeyId"), "kmsKeyId cannot be changed"))
 	}
 
-	if fi.BoolValue(obj.EncryptedVolume) != fi.BoolValue(old.EncryptedVolume) {
-		allErrs = append(allErrs, field.Forbidden(fp.Child("encryptedVolume"), "encryptedVolume cannot be changed"))
-	}
-
```

Comment:
@h3poteto I think would be enough to remove the checks for encryption, key and, we are are here, also size.

Reply:
In this case, I think that it is enough to delete the validation for encryptedVolume. Do I need to change the key and size validation?

Reply:
You should remove those too, as key is related to encryption and size can be changed now, so would be nice to not do another PR for it.

Reply:
Btw, this will work with other commands like `create -f` or `replace -f`, this is why better to not mention the `edit` comand.

Reply:
I see, I will remove those.

Reply:
Thanks. Please also squash and should be good to merge.

Reply:
👍